tests/5.1/teams/test_target_set_num_teams.c

```diff
@@ -0,0 +1,51 @@
+//===--------------------- test_target_set_num_teams.c ----------------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+//
+// This test uses the omp_set_num_teams utility to set the default number of
+// teams allocated for team clauses. The test runs teams and checks to see if
+// the number allocated is correct and reflective of the new default set. It
+// then checks to see if an explicitly stated num_teams clause overrides
+// the omp_set_num_teams properly.
+//
+//===------------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+int main() {
+
+	OMPVV_TEST_OFFLOADING;
+
+	int errors = 0; 
+	int num_teams = 0;
+
+        #pragma omp target
+        {
+	  omp_set_num_teams(4);
+        }
```

Comment:
With clang on Frontier
```
ld.lld: error: undefined symbol: omp_set_num_teams
>>> referenced by /tmp/test_target_set_num_teams.c.o.amdgcn-amd-amdhsa.gfx90a-0321f1.o:(__omp_offloading_7c_1476ac9d_main_l25)
>>> referenced by /tmp/test_target_set_num_teams.c.o.amdgcn-amd-amdhsa.gfx90a-0321f1.o:(__omp_offloading_7c_1476ac9d_main_l25)
clang: error: ld.lld command failed with exit code 1 (use -v to see invocation)

```

From the specification 5.1, page 399:
3.4.3 omp_set_num_teams
Restrictions
Restrictions to the omp_set_num_teams routine are as follows:
• The routine may not be called from within a parallel region that is not the implicit parallel region
that surrounds the whole OpenMP program.

Reply:
It is not being called from a parallel region. 
It is possible that the clang llvm does not support it yet.